web/src/message_edit.js

```diff
@@ -855,6 +856,13 @@ export function save_inline_topic_edit($row) {
 }
 
 export function save_message_row_edit($row) {
+    const $banner_container = compose_banner.get_compose_banner_container(
+        $row.find(".message_edit_form textarea"),
+    );
+    const stream_id = Number.parseInt(
+        rows.get_message_recipient_header($row).attr("data-stream-id"),
+        10,
+    );
```

Comment:
I'm concerned that this will throw an exception when editing a private message.

Reply:
If the attribute `data-stream-id` does not exist, calling `attr("data-stream-id")` will return `undefined`. Subsequently, passing `undefined` to `parseInt` will result in the return of `NaN`. This approach looks viable.